src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
```

Comment:
Usually, developers use `cn`

```suggestion
import cn from 'classnames';
```